tests/attestation_cert_apis/host/host.cpp

```diff
@@ -92,16 +94,17 @@ void run_test(oe_enclave_t* enclave, int test_type)
 
     {
         // for testing purpose, output the whole cer in DER format
-        char filename[80];
+        char filename[FILENAME_LENGTH];
         FILE* file = NULL;
 
-        sprintf(
+        sprintf_s(
             filename,
+            FILENAME_LENGTH,
             "./cert_%s.der",
             test_type == TEST_RSA_KEY ? "rsa" : "ec");
         OE_TRACE_INFO(
             "Host: Log quote embedded certificate to file: [%s]\n", filename);
-        file = fopen(filename, "wb");
+        fopen_s(&file, filename, "wb");
         fwrite(cert, 1, cert_size, file);
```

Comment:
A comment on pre-existing code: this isn't checking for fopen failure, which seems suspect.  This PR is ok though since it's no worse than before.